examples/mock-identity/mockoon-bbidentity.json

```diff
@@ -3218,31 +3218,35 @@
     },
     {
       "type": "route",
-      "uuid": "d4170fb5-331e-4065-8fc0-5a377bb57307"
+      "uuid": "1eb60f0f-7bd8-41bc-856e-6ab2eae647fc"
     },
     {
       "type": "route",
-      "uuid": "432e2b24-4bf0-4573-9cac-f5bd555a838a"
+      "uuid": "0eb3794b-3f18-4007-acea-a12a6206b500"
     },
     {
       "type": "route",
       "uuid": "4370f082-8742-4e17-8a05-0a5643f69e1e"
     },
     {
       "type": "route",
-      "uuid": "8f51c09f-816b-4137-a8aa-8e30bd56a2f7"
+      "uuid": "67587670-d11a-4e8e-a3ef-5cceb0eeed64"
     },
     {
       "type": "route",
-      "uuid": "432e2b24-4bf0-4573-9cac-f5bd555a838a"
+      "uuid": "7ab6c815-e9ae-4e3a-b04e-54239ab25d9f"
     },
     {
       "type": "route",
-      "uuid": "5674c6e6-5e80-4f0b-9bcb-30416f76d9a7"
+      "uuid": "d4170fb5-331e-4065-8fc0-5a377bb57307"
```

Comment:
Is it necessary to change every uuid in this file? This will generate conflicts

Reply:
yes, because we have randomly generated uuid's in our mockoon app I believe, no idea if there is an option to set the default value to the id.